playbooks/common/openshift-node/config.yml

```diff
@@ -69,7 +68,6 @@
   roles:
   - role: os_firewall
   - role: openshift_node
-    openshift_ca_host: "{{ groups.oo_first_master.0 }}"
```

Comment:
Cool, `openshift_ca_host` needed only when the `openshift_node_certificates` is run.